src/ethereum/frontier/vm/gas.py

```diff
@@ -25,6 +25,8 @@
 GAS_LOW = U256(5)
 GAS_MID = U256(8)
 GAS_EXPONENTIATION = U256(10)
+GAS_SHA3 = U256(30)
+GAS_SHA3_WORD = U256(6)
```

Comment:
I think to make this exceedingly obvious, we should use `KECCAK256` and not `SHA3` since we technically use the unmodified version of the algorithm.

Reply:
I agree, but this is how the constant is specified in the yellowpaper. Do you think renaming it would confuse the readers?

Reply:
![5gitjb](https://user-images.githubusercontent.com/57262657/125730721-01f00f94-1e2d-482e-b612-a9d97397e027.jpg)

Reply:
Love the meme. Will change it.